images/peer/Dockerfile

```
@@ -7,6 +7,10 @@ ARG ALPINE_VER
 
 FROM alpine:${ALPINE_VER} as peer-base
 RUN apk add --no-cache tzdata
+# set up nsswitch.conf for Go's "netgo" implementation
+# - https://github.com/golang/go/blob/go1.9.1/src/net/conf.go#L194-L275
+# - docker run --rm debian:stretch grep '^hosts:' /etc/nsswitch.conf
+RUN [ ! -e /etc/nsswitch.conf ] && echo 'hosts: files dns' > /etc/nsswitch.conf
```

I tend to agree with Matt - I think we can just go with

```suggestion
RUN echo 'hosts: files dns' > /etc/nsswitch.conf
```

given the whole point of this is to allow people to use `/etc/hosts` to override DNS resolution